vercel/resource_project.go

```diff
@@ -206,11 +210,48 @@ At this time you cannot use a Vercel Project resource with in-line ` + "`environ
 							stringLengthBetween(1, 72),
 						},
 					},
-					"protect_production": schema.BoolAttribute{
-						Description: "If true, production deployments will also be protected",
-						Optional:    true,
-						Computed:    true,
-						Default:     booldefault.StaticBool(false),
+					"deployment_type": schema.StringAttribute{
+						Required:      true,
+						Description:   "The deployment environment to protect. Must be one of `standard_protection`, `all_deployments`, or `only_preview_deployments`.",
+						PlanModifiers: []planmodifier.String{stringplanmodifier.UseStateForUnknown()},
+						Validators: []validator.String{
+							stringOneOf("standard_protection", "all_deployments", "only_preview_deployments"),
+						},
+					},
+				},
+			},
+			"trusted_ips": schema.SingleNestedAttribute{
+				Description: "Ensures only visitors from an allowed IP address can access your deployment.",
+				Optional:    true,
+				Attributes: map[string]schema.Attribute{
+					"addresses": schema.SetAttribute{
+						Description:   "The allowed IP addressses and CIDR ranges with optional descriptions.",
+						Required:      true,
+						PlanModifiers: []planmodifier.Set{setplanmodifier.UseStateForUnknown()},
+						ElementType: types.ObjectType{
+							AttrTypes: map[string]attr.Type{
+								"value": types.StringType,
+								"note":  types.StringType,
```

Comment:
This seems to make `note` required, but it's not required in the UI 🤔. Probably fine though.

Reply:
I updated it, and also marked `value` as sensitive